<ui version="4.0" >
 <class>SecondIdentifierLanguagePage</class>
 <widget class="QWidget" name="SecondIdentifierLanguagePage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>490</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>So far your document contains:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="firstLanguageName" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip" >
      <string>To change the first selection go back one page.</string>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>513</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Please select a second language (can be the same as before):</string>
     </property>
     <property name="buddy" >
      <cstring>languageComboBox</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="languageComboBox" >
     <property name="toolTip" >
      <string>The language of the second column of vocabulary.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Second title:</string>
     </property>
     <property name="buddy" >
      <cstring>identifierNameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="identifierNameLineEdit" >
     <property name="toolTip" >
      <string>This does not have to be a language. Anything goes: Definition, Chinese simplified or anything else you want to learn.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
